Description: don't use __proto__
Author: Yadd <yadd@debian.org>
Forwarded: no
Last-Update: 2022-09-17

--- a/lib/colors.js
+++ b/lib/colors.js
@@ -89,7 +89,7 @@
   builder._styles = _styles;
   // __proto__ is used because we must return a function, but there is
   // no way to create a function with a different prototype.
-  builder.__proto__ = proto;
+  Object.setPrototypeOf(builder, proto);
   return builder;
 }
 
